test/LeanCode.IntegrationTests/TimestampTzTests.cs

```diff
@@ -0,0 +1,67 @@
+using LeanCode.DomainModels.Model;
+using LeanCode.IntegrationTests.App;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace LeanCode.IntegrationTests;
+
+[System.Diagnostics.CodeAnalysis.SuppressMessage("?", "CA1001", Justification = "Disposed with `IAsyncLifetime`.")]
+public class TimestampTzTests : IAsyncLifetime
+{
+    private readonly TestApp app;
+
+    public TimestampTzTests()
+    {
+        app = new TestApp();
+    }
+
+    [PostgresFact]
+    public async Task TimestampTz_local_timestamp_access_is_correctly_translated()
+    {
+        using var scope = app.Services.CreateScope();
+        var dbContext = scope.ServiceProvider.GetRequiredService<TestDbContext>();
+        var today = DateOnly.FromDateTime(DateTime.UtcNow);
+
+        var meeting1 = new Meeting
+        {
+            Id = Guid.NewGuid(),
+            Name = "First",
+            StartTime = new(today.ToDateTime(new(10, 0), DateTimeKind.Utc), "Asia/Tokyo")
+        };
+
+        var meeting2 = new Meeting
+        {
+            Id = Guid.NewGuid(),
+            Name = "Second",
+            StartTime = new(today.ToDateTime(new(14, 0), DateTimeKind.Utc), "America/Los_Angeles")
+        };
+
+        dbContext.Meetings.Add(meeting1);
+        dbContext.Meetings.Add(meeting2);
+
+        await dbContext.SaveChangesAsync();
+
+        dbContext.ChangeTracker.QueryTrackingBehavior = QueryTrackingBehavior.TrackAll;
+
+        var orderedByLocalQuery = dbContext.Meetings.OrderBy(m => m.StartTime.LocalTimestampWithoutOffset);
+
+        Assert.Contains(
```

Comment:
I wonder if we can get inspired by other providers and test it in a better fashion. Technically, you don't even need a running postgres to check if the sql is generated correctly,

Reply:
You are right, just like we don't need running database just to check if all migrations were generated, and yet we do run such checks as integration tests because it's simpler.

Should we prepare unit tests to reference `Npgsql` and build `DbContext` just to execute a single assert?

Reply:
Just thinking out loud - I just don't know how others do it.